src/app/tests/suites/certification/Test_TC_MTRID_3_1.yaml

```diff
@@ -0,0 +1,832 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+name: X.2.1. [TC-MTRID-3.1] Subscription Report Verification with DUT as Server
```

Comment:
From what I can understand this is mostly a "manual" test? Would it be worth implementing it in python instead a try to automate it? I do not think we should do manual tests at this point.